### ISSUE TITLE
Empty response from Tiktok

### ISSUE THREAD

**Issue by Bearbeitung** [samirelanduk/tiktok-save#16]
Is this script still working in 2024? When I try to download my liked videos right now I get empty responses.

`TikTokApi.exceptions.EmptyResponseError: Empty response from Tiktok`

**Comment by samirelanduk** [comment]
The TikTok API has changed since I created this tool. It may still be possible to get it working, but unfortunately I just don't have the time at the moment.

**Comment by Trevaweb** [comment]
I have spent some time trying to get this working again so just wanted to document the issues I came across that kept me from doing so. I got most of the functionality working again except for the actual video download portion. The newest version of the TikTop API has removed the functionality to download videos using it. Its intended to use video.byte() to download videos but it has not yet been implemented and has been intended to be implemented for at least a year or more. This is documented [here](https://github.com/davidteather/TikTok-Api/issues/1040) as well as in some other threads but its recommended to use yt-dlp as a workaround. While trying to get yt-dlp to work I came across some issues where they haven't released an new version in a very long time which was causing problems for me and there was also a bug with the code for downloading tiktok videos(I'm a few weeks out from last working on this so the details are fuzzy). I came to the conclusion that until either yt-dlp was updated/fixed or the TikTok API implemented the byte() function that there wasn't much I could do to get this running. I will probably take a look at this again in the future sometime as I'm very interested in getting my saved videos downloaded.
